proto/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name         = "ibc-proto"
-version      = "0.18.0"
+version      = "0.19.0"
```

Comment:
Doesn't seem like there have been breaking changes in crates other than `ibc` and `ibc-relayer-cli`, so should we continue to bump versions for all crates?

Reply:
The `ibc-proto` crate contains [these changes](https://github.com/informalsystems/ibc-rs/pull/2278) which are indeed not breaking but are also a sizable feature so to me it makes sense to bump its major version as well. What do you think?

Reply:
Right, I agree it makes sense to bump the version. :+1: What do you suggest regarding the other worksapce crates?

Reply:
For now let's bump them all, we can figure out a proper process for this after the rc0 release, in the form of an ADR.